### PR TITLE
Docs/swagger

### PR DESCRIPTION
**Pull request by xynazog**
This PR is raised in order to merge the swagger doc code. This is connected with issue #110 
This merge would have all the code except searching a wishlist.

### CODE REVIEW

File: app/server.py [hunk 1]
```diff
@@ -29,12 +34,70 @@ def index():
 @app.route('/wishlists',methods=['POST'])
 def add_wishlist():
 	"""
-	The route for adding new wishlists, specified by userID and name of the wishlist. You can check
-	the POST requests using CURL.
-	Example: curl -i -H 'Content-Type: application/json' -X POST -d '{"name":"Xynazog","user_id":123}' http://127.0.0.1:5000/wishlists
-	H is for headers, X is used to specify HTTP Method, d is used to pass a message.
-	In location headers, if _external set to True, an absolute URL is generated. 
-	"""
+    Creates a Wishlist
```
Comment (pinocchiochang) [inline app/server.py:37]: Can you please reformat the comments?

**Review by pinocchiochang — CHANGES_REQUESTED** [review]
Just commented.

File: app/server.py [hunk 2]
```diff
@@ -50,11 +113,74 @@ def add_wishlist():
 @app.route('/wishlists/<int:wishlist_id>/items',methods=['POST'])
 def add_item_to_wishlist(wishlist_id):
 	"""
```
Comment (pinocchiochang) [inline app/server.py:115]: Same here.

File: app/server.py [hunk 3]
```diff
@@ -75,9 +201,48 @@ def add_item_to_wishlist(wishlist_id):
 @app.route('/wishlists', methods=['GET'])
 def wishlists():
 	"""
```
Comment (pinocchiochang) [inline app/server.py:203]: Format

**Review by pinocchiochang — APPROVED** [review]
Looks Good Now !!!

**Review by pinocchiochang — APPROVED** [review]
cool

**Review by palakbhasin — APPROVED** [review]
Works for me

**Pull request merged**